bpf/lib/egress_policies.h

```diff
@@ -371,6 +371,7 @@ srv6_create_state_entry(struct __ctx_buff *ctx)
 
 		if (map_update_elem(&SRV6_STATE_MAP6, inner_ips, outer_ips, 0) < 0)
 			return DROP_INVALID;
+		break;
```

Comment:
These seem like genuine mistakes, maybe split them into a separate commit with a Fixes tag?

Reply:
Absolutely, I meant to do that in the first place :facepalm:. Thanks! Will update.